modules/cli/src/main/java/org/apache/ignite/cli/deprecated/builtins/node/NodeManager.java

```diff
@@ -136,6 +137,9 @@ public RunningNode start(
             if (srvCfg != null) {
                 cmdArgs.add("--config");
                 cmdArgs.add(srvCfg.toAbsolutePath().toString());
+            } else if (srvCfgStr != null) {
+                cmdArgs.add("--configStr");
```

Comment:
`--configStr` does not look descriptive to be honest :) By the way, do we use camel case for commands? Perhaps it should be named `config-str`, for instance see at `work-dir`.

Reply:
`--config-file` and `--config-string` maybe?

Reply:
Perhaps, `config-overrides` would cleaner. What do you think?

Reply:
Looks like I was wrong. My last comment does not make sense. Please ignore.